src/calendar-heatmap.component.js

```diff
@@ -1783,13 +1770,13 @@ export class CalendarHeatmap extends Component {
     let minutes = Math.floor((seconds - hours * 3600) / 60);
     let time = '';
     if (hours > 0) {
-      time += hours === 1 ? '1 hour ' : hours + ' hours ';
+      time = hours === 1 ? `${time}1 hour ` : `${time}${hours} hours `;
     }
     if (minutes > 0) {
-      time += minutes === 1 ? '1 minute' : minutes + ' minutes';
+      time = minutes === 1 ? `${time}1 minute` : `${time}${minutes} minutes`;
```

Comment:
it is okay for now, but make a new issue and look into https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/PluralRules/PluralRules